redis/src/commands/mod.rs

```diff
@@ -1109,6 +1109,13 @@ implement_commands! {
         cmd("ACL").arg("DELUSER").arg(usernames)
     }
 
+    /// Simulate the execution of a given command by a given user.
+    #[cfg(feature = "acl")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "acl")))]
+    fn acl_dryrun<P: ToRedisArgs>(username: P, command: P, args: &'a [P]) {
```

Comment:
please give separate types for the arguments, to allow the user more freedom, and make sure they all implement ToRedisArgs

Reply:
Thank you for commenting on my draft. I'll fix it now.

Reply:
There may be one problem here though. Your Github Action checks for redis versions 7 and 6. However, this command is available from version 7. Well, let me correct the command first.

Reply:
You can check the db's version and exit realty if it doesn't support the command. I'm not next to a computer right now, but look for is_major_version in tests

Reply:
Thanks!